### PR TITLE
oohmm, VillagerTrade added monkaDie

### PR DESCRIPTION
**Pull request by many1337**

### CODE REVIEW

File: src/CortexPE/EventListener.php [hunk 1]
```diff
@@ -355,4 +376,74 @@ public function onTeleport(EntityTeleportEvent $ev){
...
+        }
+    }
+
+    /**
+     * @param Item $buyA
+     * @param Item $sell
+     * @param Item|null $buyB
+     * @return CompoundTag
+     */
+    public function makeRecipe(Item $buyA, Item $sell, Item $buyB = null){
```
Comment (NycuRO) [inline src/CortexPE/EventListener.php]: make multiple recipes addon

Comment (many1337) [reply src/CortexPE/EventListener.php]: Okay 🤔

**Comment by NycuRO** [comment]
This still works?

**Comment by many1337** [comment]
> This still works? Give me a sign with a proof. If yes, i will accept it.

i dont have time to developing but i try fix in February

**Comment by many1337** [comment]
[THIS](https://github.com/many1337/VillagerTradeAPI)

**Pull request closed without merging**